Assets/Thirdweb/Core/Scripts/Wallets/ThirdwebLocalHdWallet.cs

```diff
@@ -0,0 +1,90 @@
+using System.Numerics;
+using System.Threading.Tasks;
+using Nethereum.Web3;
+using Nethereum.Web3.Accounts;
+
+namespace Thirdweb.Wallets
+{
+    public class ThirdwebLocalHdWallet: IThirdwebWallet
+    {
+        private Account _account;
+        private Web3 _web3;
+        private readonly WalletProvider _provider;
+        private readonly WalletProvider _signerProvider;
+
+        public ThirdwebLocalHdWallet()
+        {
+            _account = null;
+            _web3 = null;
+            // I guess, we can use the local wallet provider here to avoid breaking conditions that rely on the local wallet provider
+            _provider = WalletProvider.LocalWallet;
```

Comment:
This should say WalletProvider.LocalHdWallet, we can update special wallet cases elsewhere